workers/social/lib/social/models/group/index.coffee

```diff
@@ -1705,6 +1705,12 @@ module.exports = class JGroup extends Module
         JName.one { name: slug }, (err, name) ->
           removeHelper name, err, next, 'JName'
 
+      (next) ->
+        # unlink slack if anyone in this team auth slack previously
+        JUser = require '../user'
+        JUser.update { "foreignAuth.slack.#{slug}": { $exists: true } }, { $unset: { "foreignAuth.slack.#{slug}": 1 } }, (err) ->
+          kallback 'JForeignAuth', next, err
```

Comment:
we had slack integration before migration of JForeignAuth. it seems we forgot to migrate that as well, best thing to do would be migrating slack to JForeignAuth